w2/assignment2_2_2/Calculator/src/utils.cpp

```diff
@@ -0,0 +1,95 @@
+/* Create a calculator program that can do basic operations:
+ * Addition, subtraction, multiplication and division.
+ * In your program, create a user interface that asks for basic calculation operations.
+ * All calculations should be executed in a utility library.
+ */
+
+#include <vector>
+#include <iostream>
+
+double add(const std::vector<double> &vect)
+{
+    double sum{};
+    for (auto elem : vect)
+    {
+        sum += elem;
+    }
+    return sum;
+}
```

Comment:
Library might be forcing to use vectors as parameters but it does make it harder to understand that this is just a normal addition of a number to another.